linera-rpc/tests/snapshots/format__format.yaml.snap

```diff
@@ -940,6 +940,13 @@ RpcMessage:
       CrossChainRequest:
         NEWTYPE:
           TYPENAME: CrossChainRequest
+Secp256k1PublicKey:
+  NEWTYPESTRUCT:
+    TUPLEARRAY:
+      CONTENT: U8
+      SIZE: 33
+Secp256k1Signature:
+  NEWTYPESTRUCT: BYTES
```

Comment:
TODO: Serialize as array, not vector.

Reply:
Switched to serialization with compact representation (which is always 64 bytes). See [Serialize secp256k1 signatures in a compact form rather than DER.](https://github.com/linera-io/linera-protocol/pull/3377/commits/a2a3dab84dcb5139ddc210a3c2d0fdd24ced4136).